poc/gen_test_vec.py

```diff
@@ -65,3 +66,83 @@
             measurements,
             i,
         )
+
+        pine32_hmac_sha256_aes128 = Pine32HmacSha256Aes128(
+            l2_norm_bound,
+            num_frac_bits,
+            dimension,
+            chunk_length,
+            chunk_length_norm_equality,
+            num_shares,
+            num_proofs=5,
+            num_proofs_norm_equality=1,
+            num_wr_checks=num_wr_checks,
+            num_wr_successes=num_wr_successes,
+        )
+        gen_test_vec_for_vdaf(
+            TEST_VECTOR_PATH,
+            pine32_hmac_sha256_aes128,
+            None,
+            measurements,
+            2 * i,
+        )
+
+        pine32_hmac_sha256_aes128 = Pine32HmacSha256Aes128(
+            l2_norm_bound,
+            num_frac_bits,
+            dimension,
+            chunk_length,
+            chunk_length_norm_equality,
+            num_shares,
+            num_proofs=4,
```

Comment:
Suggestion: From the point of view of the API, it's a good idea to provide safe defaults wherever we can and not let the user override them. We think 4 proofs is probably too low for Field32. Thus a "safer" API for `Pine32HmacSha256Aes128` would not let the user do this.

That said, I understand the motivation for generating test vectors with fewer proofs. An alternative would be invoke the constructor for `Pine` instead and manually specify the field and XOF.

Reply:
Ok. I removed `num_proofs` from `Pine32HmacSha256Aes128`, initially I was following what we allowed in taskprov. And I will use `Pine` to create custom `Pine32HmacSha256Aes128`.